specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/SourceControls.json

```diff
@@ -171,6 +171,13 @@
               "$ref": "#/definitions/SourceControl"
             }
           },
+          {
+            "name": "x-ms-github-auxiliary",
+            "in": "header",
+            "description": "Github personal access token used for SourceControl.",
+            "required": true,
+            "type": "string"
+          },
           {
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
```

Comment:
>v3 [](http://example.com/codeflow?start=69&length=2)

update to v5 here and at all places in this file for common-types

Reply:
For consistency you should also update the rest of the parameters to v5